gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,24 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):  # pragma: NO COVER
+            # TODO: Make this performant when logging is not enabled
+
+            # TODO: Remove this condition once the minimum version of google-auth is 2.35.0
```

Comment:
File a bug for the TODO comment

Reply:
I've removed the condition so a bug is no longer needed.